appinventor/buildserver/src/com/google/appinventor/buildserver/Compiler.java

```diff
@@ -628,6 +628,8 @@ private boolean writeAndroidManifest(File manifestFile) {
         }
       }
 
+      minSdk = 22;
```

Comment:
Do not change the minSdk. This would prevent anyone with a device running a version of Android < 5.1 from being able to use App Inventor.